docs/cow-protocol/reference/core/auctions/schema.md

```diff
@@ -15,441 +18,188 @@ To avoid precision loss, some numerical literals are encoded as strings, referre
-- The lowest denomination of `BAL` (2nd entry) is 1 / 10¹⁸ , and it has a price of 5245932598960804 relative to the price of 10¹⁸ that wei has. This translates to 1 `BAL` having a price of $$\frac{5245932598960804 \cdot 10^{18}}{10^{18} \cdot 10^{18}} \approx 0.005245933$$ WETH.
+- 1 wei (3rd entry) has a price of 1000000000000000000 = 10<sup>18</sup>.
+- The lowest denomination of `USDC` (1st entry), i.e., 1 / 10<sup>6</sup> of `USDC`, has a price of 389193245742363509576247472 relative to the price of 10<sup>18</sup> that wei has. This translates, for example, to 1 `WETH` having a price of $$\frac{1000000000000000000 \cdot 10^{18}}{389193245742363509576247472 \cdot 10^{6}} \approx 2569.417$$ `USDC`.
+- The lowest denomination of `COW` (2nd entry) is 1 / 10<sup>18</sup> , and it has a price of 137298311435590. This translates, for example, to 1 `COW` having a price of $$\frac{137298311435590 \cdot 10^{18}}{389193245742363509576247472 \cdot 10^{6}} \approx 0.35277$$ USDC.
 
-### `approvals`
 
-In order to allow solvers to propose solutions that interact with contracts/pools that are not provided by the Driver, there are 2 additional sections in the solution file. The first is the "approvals" key and the second is the "interaction_data" key (covered below). We clarify that both are required fields.
+### `trades`
 
-The "approvals" key is a list where each entry consists of the following:
+This key maps to a list of all orders that were selected for execution. Each trade is a dictionary that contains the following entries:
 
-- `"spender"`: the address of the target contract that we authorize to trade some token on the settlement contract's behalf.
-- `"token"`: the address of the token that we authorize.
-- `"amount"`: a stringified integer that corresponds to the (minimum) amount we authorize the target contract to use. **We stress here** that currently the behavior is the following. If there is already sufficiently large (i.e., larger than the `"amount"` entry) allowance set by a previous settlement, then the approval is ignored altogether. On the other hand, if the current allowance is lower than `"amount"`, then the driver will set a maximum value for the approval (much larger than the actual `"amount"` specified by the entry); this allows for future gas savings.
+- `kind`: this is string of the set {"fulfillment", "jit"}, which corresponds to an order existing in the orderbook, or a just-in-time liquidity order placed by the solver, respectively.
+- `order`: in case of a "fulillment" trade, the `uid` of the order is provided here as a string. In case it is a just-in-time liquidity order, the specifications of the order are explicitly given as a dictionary; more details can be found [here](https://docs.cow.fi/cow-protocol/reference/apis/solver).
```

Comment:
```suggestion
- `order`: in case of a "fulfillment" trade, the `uid` of the order is provided here as a string. In case it is a just-in-time liquidity order, the specifications of the order are explicitly given as a dictionary; more details can be found [here](https://docs.cow.fi/cow-protocol/reference/apis/solver).
```